### PR TITLE
Remove Licence

### PR DESCRIPTION
**Pull request by Asleeepp**
### Description

This PR aims to remove the licence header from all files
god please help me my web browser keeps freezing

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/test/java/org/skriptlang/skript/test/junit/registration/package-info.java [hunk 1]
```diff
@@ -1,22 +1,2 @@
...
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
- *
- * Copyright Peter Güttinger, SkriptLang team and contributors
- * 
- * Place any and all custom syntaxes relating to the JUnit testJar in here to be exclusively ran on the test runner.
```
Comment (erenkarakal) [inline src/test/java/org/skriptlang/skript/test/junit/registration/package-info.java:19]: I don't think you should remove this line

Comment (Asleeepp) [reply src/test/java/org/skriptlang/skript/test/junit/registration/package-info.java:19]: bruh who put it in the licence????

**Review by Moderocky — CHANGES_REQUESTED** [review]
We want to keep the licence header on the `package-info.java` files.
While most files are just GPL-3, some packages are under other compatible licences, and it would be nice to keep those to prevent any confusion about what is what.

**Review by Efnilite — APPROVED** [review]

**Review by Moderocky — APPROVED** [review]

**Comment by Moderocky** [comment]
I wonder what the `+20` lines added are.

**Comment by BaeFell** [comment]
> I wonder what the `+20` lines added are.

image

**Pull request merged**